packages/wallet-service/src/api/healthcheck.ts

```diff
@@ -0,0 +1,143 @@
+import middy from "@middy/core";
+import {
+    Healthcheck,
+    HealthcheckInternalComponent,
+    HealthcheckDatastoreComponent,
+    HealthcheckHTTPComponent,
+    HealthcheckCallbackResponse,
+    HealthcheckStatus,
+} from "@hathor/healthcheck-lib";
+import { getLatestHeight } from "@src/db";
+import fullnode from "@src/fullnode";
+import { closeDbConnection, getDbConnection } from "@src/utils";
+import { APIGatewayProxyHandler } from "aws-lambda";
+import { getRedisClient, ping } from "@src/redis";
+
+const mysql = getDbConnection();
+
+const checkDatabaseHeight: HealthcheckCallbackResponse = async () => {
+    try {
+        const [currentHeight, fullnodeStatus] = await Promise.all([
+            getLatestHeight(mysql),
+            fullnode.getStatus()
+        ]);
+
+        const currentFullnodeHeight = fullnodeStatus["dag"]["best_block"]["height"];
```

Comment:
s/"/'/g

```suggestion
        const currentFullnodeHeight = fullnodeStatus['dag']['best_block']['height'];
```

Reply:
Do we have an automatted formatter for the project? I think it would be the best solution to keep a style standard.

Reply:
Fixed in https://github.com/HathorNetwork/hathor-wallet-service-sync_daemon/pull/110/commits/555184bdbaf034a17b2337e868ef0dc5476fd4d9